user-console/src/main/java/org/pentaho/mantle/client/ui/xul/MantleXul.java

```diff
@@ -170,6 +180,28 @@ public void onResponseReceived( Request request, Response response ) {
       // showError(e);
     }
 
+    final String editUrl = contextURL + "api/repo/files/canEdit"; //$NON-NLS-1$
```

Comment:
Afaik, the `//$NON-NLS-1$` indications are no longer used.

Reply:
will update